docs/pipeline.rst

```diff
@@ -66,6 +66,19 @@ pushes, this is all handled by the pipeline, and is subject to change according
     #. If any staging tests fail above, check retries, e.g. `bedrock_test_stage_eu_west`_
 #. Tag and push the deployment by running ``bin/tag-release.sh --push``
 
+.. note::
+
+    By default the ``tag-release.sh`` script will push to the ``origin`` git remote. If you'd
+    like for it to push to a different remote name you can either pass in a ``-r`` or
+    ``--remote`` argument, or set the ``MOZ_GIT_REMOTE`` environment variable. So the following
```

Comment:
Is it possible to have `MOZ_GIT_REMOTE` picked up from `.env`? If not, I would probably just change the name of my remote to match the default (to avoid having to specify each time), which is maybe what we want anyway.